apps/src/templates/CollapsibleSection.tsx

```diff
@@ -0,0 +1,55 @@
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import Typography from '@cdo/apps/componentLibrary/typography/Typography';
+import React, {useState} from 'react';
+import moduleStyles from './collapsible-section.module.scss';
+import {
+  SemanticTag,
+  VisualAppearance,
+} from '@cdo/apps/componentLibrary/typography/types';
+
+const CollapsibleSection: React.FunctionComponent<{
+  title: string;
+  titleSemanticTag?: SemanticTag;
+  titleVisualAppearance?: VisualAppearance;
+  children: React.ReactNode;
+  initiallyCollapsed?: boolean;
+  collapsedIcon?: string;
+  expandedIcon?: string;
+}> = ({
+  title,
+  titleSemanticTag = 'p',
+  titleVisualAppearance = 'body-one',
```

Comment:
What do you think of instead passing in the typography class name (`BodyOneText`) in place of these? Curious what @levadadenys thinks is the best way to pass this information in here

Reply:
I'll defer to @levadadenys . I initially thought of this approach, but it seems like maybe I'd need to add a type in `typography/types.ts` for the Typography types defined in `TypographyElements.
Something like:
```
type TypographyElement = Heading1 | Heading2 | ... | BodyOneText | ...;
```
Unsure about this. Thanks!

Reply:
I think @fisher-alice used the best possible approach here from what I see in the code. If we assume title can be rendered with different sematicTag and VisualAppearance - that's how it's intended to be done.

Passing class name instead won't solve different semantic tags and also will only work if we'll be passing typographyModuleStyles[visualAppearance] which might be harder to read and understand, plus className of typography elements are in moduleStyles, meaning we can't just pass a 'body-one' className string since moduleStyles classNames are parsed into some 'coded' classNames.

Two things we might try here I think:
1) 
```
 import {
   SemanticTag as TypographyElementSematicTag,
   VisualAppearance as TypographyElementVisualAppearance,
 } from '@cdo/apps/componentLibrary/typography/types';
 ```
  for better readability
2) If we're sure that semanticTag will always be 'p' - don't pass it as a prop

Reply:
Thanks Denys!
I will update to your first suggestion above.
And `semanticTag` will not always be 'p' so will keep it as a prop. For the presentation view of model card, the `semanticTag` is assigned 'h6'.